Psycopg version of queue library

dagmawig opened the pull request:
This is a demo for a python library version of the pub/sub system.
It passes the end to end tests implemented on the dev database.

dagmawig committed:
feat: create virtual env

dagmawig committed:
feat: add fetch function and e2e test

dagmawig committed:
feat: add publish and subscribe functions and e2e test

dagmawig committed:
chore: update README file

dadiorchen reviewed (changes requested):
@dagmawig I think this one is good, except, please remove those file in:
vscode
bin file
env file that can be generated by dev

dagmawig committed:
chore: remove .vscode folder

dagmawig commented:
> @dagmawig I think this one is good, except, please remove those file in: vscode bin file env file that can be generated by dev

I have removed the .vscode folder and added it to .gitignore.

dagmawig committed:
chore: remove venv folder, update README

dagmawig commented:
Snapshot of test result.
![image](https://github.com/user-attachments/assets/b4184ad0-9e7d-47bf-9be4-1000fb2de42c)

dagmawig committed:
chore: remove __pycache__ folders
